### PR TITLE
Bump RLL and improve logging

### PR DESCRIPTION
**Pull request by Drapersniper**
- bump RLL
- logging changes
- player.channel.guild->player.guild

What to test:
 - Nothing, no changes here affect existing functionality, start the bot with `--debug` and run through a few commands just to make sure logging doesnt break with new formatting

### CODE REVIEW

File: redbot/cogs/audio/apis/global_db.py [hunk 1]
```diff
@@ -108,14 +110,16 @@ async def get_spotify(self, title: str, author: Optional[str]) -> dict:
                     search_response = await r.json(loads=json.loads)
                     if IS_DEBUG and "x-process-time" in r.headers:
                         log.debug(
-                            f"GET/spotify || Ping {r.headers.get('x-process-time')} || "
-                            f"Status code {r.status} || {title} - {author}"
+                            "GET/spotify || Ping %s || Status code %d || %s - %s",
+                            r.headers.get("x-process-time"),
+                            r.status,
+                            title.author,
```
Comment (Jackenmen) [inline redbot/cogs/audio/apis/global_db.py]:
```suggestion
                            "GET/spotify || Ping %s || Status code %d || %s - %s",
                            r.headers.get("x-process-time"),
                            r.status,
                            title,
                            author,
```

File: redbot/cogs/audio/apis/interface.py [hunk 2]
```diff
@@ -149,19 +149,19 @@ async def run_tasks(self, ctx: Optional[commands.Context] = None, message_id=Non
...
+                    log.debug("Running database writes for %d (%s)", lock_id, lock_author)
                 try:
                     tasks = self._tasks[lock_id]
                     tasks = [self.route_tasks(a, tasks[a]) for a in tasks]
                     await asyncio.gather(*tasks, return_exceptions=False)
                     del self._tasks[lock_id]
                 except Exception as exc:
                     debug_exc_log(
-                        log, exc, f"Failed database writes for {lock_id} ({lock_author})"
+                        log, exc, "Running database writes for %d (%s)", lock_id, lock_author
```
Comment (Jackenmen) [inline redbot/cogs/audio/apis/interface.py]: I think you didn't mean to change this?
```suggestion
                        log, exc, "Failed database writes for %d (%s)", lock_id, lock_author
```

File: redbot/cogs/audio/apis/interface.py [hunk 3]
```diff
@@ -149,19 +149,19 @@ async def run_tasks(self, ctx: Optional[commands.Context] = None, message_id=Non
...
                     )
                 else:
                     if IS_DEBUG:
-                        log.debug(f"Completed database writes for {lock_id} ({lock_author})")
+                        log.debug("Running database writes for %d (%s)", lock_id, lock_author)
```
Comment (Jackenmen) [inline redbot/cogs/audio/apis/interface.py]: I think you didn't mean to change this?
```suggestion
                        log.debug("Completed database writes for %d (%s)", lock_id, lock_author)
```

File: redbot/cogs/audio/apis/interface.py [hunk 4]
```diff
@@ -248,7 +248,9 @@ async def fetch_spotify_query(
                             {"track": track_info}
                         )
                     except Exception as exc:
-                        debug_exc_log(log, exc, f"Failed to fetch {track_info} from YouTube table")
+                        debug_exc_log(
+                            log, exc, "Failed to fetch %s from YouTube table", track_info
```
Comment (Jackenmen) [inline redbot/cogs/audio/apis/interface.py]:
```suggestion
                            log, exc, "Failed to fetch %r from YouTube table", track_info
```

File: redbot/cogs/audio/apis/interface.py [hunk 5]
```diff
@@ -513,7 +515,9 @@ async def spotify_enqueue(
...
```
Comment (Jackenmen) [inline redbot/cogs/audio/apis/interface.py]:
```suggestion
                            log, exc, "Failed to fetch %r from YouTube table", track_info
```

File: redbot/cogs/audio/apis/interface.py [hunk 6]
```diff
@@ -621,7 +625,7 @@ async def spotify_enqueue(
                 ):
                     has_not_allowed = True
                     if IS_DEBUG:
-                        log.debug(f"Query is not allowed in {ctx.guild} ({ctx.guild.id})")
+                        log.debug("Query is not allowed in %s (%d)", ctx.guild, ctx.guild.id)
```
Comment (Jackenmen) [inline redbot/cogs/audio/apis/interface.py]:
```suggestion
                        log.debug("Query is not allowed in %r (%d)", ctx.guild.name, ctx.guild.id)
```

File: redbot/cogs/audio/apis/interface.py [hunk 7]
```diff
@@ -748,7 +752,7 @@ async def fetch_from_youtube_api(
             try:
                 (val, update) = await self.local_cache_api.youtube.fetch_one({"track": track_info})
             except Exception as exc:
-                debug_exc_log(log, exc, f"Failed to fetch {track_info} from YouTube table")
+                debug_exc_log(log, exc, "Failed to fetch %s from YouTube table", track_info)
```
Comment (Jackenmen) [inline redbot/cogs/audio/apis/interface.py]:
```suggestion
                debug_exc_log(log, exc, "Failed to fetch %r from YouTube table", track_info)
```

File: redbot/cogs/audio/apis/interface.py [hunk 8]
```diff
@@ -813,11 +817,11 @@ async def fetch_track(
                     {"query": query_string}
                 )
             except Exception as exc:
-                debug_exc_log(log, exc, f"Failed to fetch '{query_string}' from Lavalink table")
+                debug_exc_log(log, exc, "Failed to fetch '%s' from Lavalink table", query_string)
```
Comment (Jackenmen) [inline redbot/cogs/audio/apis/interface.py]:
```suggestion
                debug_exc_log(log, exc, "Failed to fetch %r from Lavalink table", query_string)
```

File: redbot/cogs/audio/apis/interface.py [hunk 9]
```diff
@@ -813,11 +817,11 @@ async def fetch_track(
...
 
             if val and isinstance(val, dict):
                 if IS_DEBUG:
-                    log.debug(f"Updating Local Database with {query_string}")
+                    log.debug("Updating Local Database with %s", query_string)
```
Comment (Jackenmen) [inline redbot/cogs/audio/apis/interface.py]:
```suggestion
                    log.debug("Updating Local Database with %r", query_string)
```

File: redbot/cogs/audio/apis/interface.py [hunk 10]
```diff
@@ -851,7 +855,7 @@ async def fetch_track(
                     valid_global_entry = True
                 if valid_global_entry:
                     if IS_DEBUG:
-                        log.debug(f"Querying Global DB api for {query}")
+                        log.debug("Querying Global DB api for %s", query)
```
Comment (Jackenmen) [inline redbot/cogs/audio/apis/interface.py]:
```suggestion
                        log.debug("Querying Global DB api for %r", query)
```

File: redbot/cogs/audio/apis/interface.py [hunk 11]
```diff
@@ -870,7 +874,7 @@ async def fetch_track(
             valid_global_entry = False
         else:
             if IS_DEBUG:
-                log.debug(f"Querying Lavalink api for {query_string}")
+                log.debug("Querying Lavalink api for %s", query_string)
```
Comment (Jackenmen) [inline redbot/cogs/audio/apis/interface.py]:
```suggestion
                log.debug("Querying Lavalink api for %r", query_string)
```

File: redbot/cogs/audio/apis/interface.py [hunk 12]
```diff
@@ -923,13 +927,14 @@ async def fetch_track(
                 debug_exc_log(
                     log,
                     exc,
-                    f"Failed to enqueue write task for '{query_string}' to Lavalink table",
+                    "Failed to enqueue write task for '%s' to Lavalink table",
```
Comment (Jackenmen) [inline redbot/cogs/audio/apis/interface.py]:
```suggestion
                    "Failed to enqueue write task for %r to Lavalink table",
```

File: redbot/cogs/audio/apis/interface.py [hunk 13]
```diff
@@ -989,35 +992,32 @@ async def autoplay(self, player: lavalink.Player, playlist_api: PlaylistWrapper)
                     query_obj=query,
                 ):
                     if IS_DEBUG:
-                        log.debug(
-                            "Query is not allowed in "
-                            f"{player.channel.guild} ({player.channel.guild.id})"
-                        )
+                        log.debug("Query is not allowed in %s (%d)", player.guild, player.guild.id)
```
Comment (Jackenmen) [inline redbot/cogs/audio/apis/interface.py]:
```suggestion
                        log.debug("Query is not allowed in %r (%d)", player.guild.name, player.guild.id)
```

File: redbot/cogs/audio/apis/spotify.py [hunk 14]
```diff
@@ -102,7 +102,7 @@ async def get(
         async with self.session.request("GET", url, params=params, headers=headers) as r:
             data = await r.json(loads=json.loads)
             if r.status != 200:
-                log.debug(f"Issue making GET request to {url}: [{r.status}] {data}")
+                log.debug("Issue making GET request to %s: [%d] %s", url, r.status, data)
```
Comment (Jackenmen) [inline redbot/cogs/audio/apis/spotify.py]:
```suggestion
                log.debug("Issue making GET request to %r: [%d] %r", url, r.status, data)
```

File: redbot/cogs/audio/apis/spotify.py [hunk 15]
```diff
@@ -156,7 +156,7 @@ async def post(
         async with self.session.post(url, data=payload, headers=headers) as r:
             data = await r.json(loads=json.loads)
             if r.status != 200:
-                log.debug(f"Issue making POST request to {url}: [{r.status}] {data}")
+                log.debug("Issue making POST request to %s: [%d] %s", url, r.status, data)
```
Comment (Jackenmen) [inline redbot/cogs/audio/apis/spotify.py]:
```suggestion
                log.debug("Issue making POST request to %r: [%d] %r", url, r.status, data)
```

File: redbot/cogs/audio/core/commands/player.py [hunk 16]
```diff
@@ -290,7 +290,7 @@ async def command_bumpplay(
             query_obj=query,
         ):
             if IS_DEBUG:
-                log.debug(f"Query is not allowed in {ctx.guild} ({ctx.guild.id})")
+                log.debug("Query is not allowed in %s (%d)", ctx.guild, ctx.guild.id)
```
Comment (Jackenmen) [inline redbot/cogs/audio/core/commands/player.py]:
```suggestion
                log.debug("Query is not allowed in %r (%d)", ctx.guild.name, ctx.guild.id)
```

File: redbot/cogs/audio/core/commands/player.py [hunk 17]
```diff
@@ -834,7 +832,7 @@ async def _search_menu(
                         query_obj=query,
                     ):
                         if IS_DEBUG:
-                            log.debug(f"Query is not allowed in {ctx.guild} ({ctx.guild.id})")
+                            log.debug("Query is not allowed in %s (%d)", ctx.guild, ctx.guild.id)
```
Comment (Jackenmen) [inline redbot/cogs/audio/core/commands/player.py]:
```suggestion
                            log.debug("Query is not allowed in %r (%d)", ctx.guild.name, ctx.guild.id)
```

File: redbot/cogs/audio/core/commands/playlists.py [hunk 18]
```diff
@@ -1525,7 +1525,7 @@ async def command_playlist_start(
                         query_obj=query,
                     ):
                         if IS_DEBUG:
-                            log.debug(f"Query is not allowed in {ctx.guild} ({ctx.guild.id})")
+                            log.debug("Query is not allowed in %s (%d)", ctx.guild, ctx.guild.id)
```
Comment (Jackenmen) [inline redbot/cogs/audio/core/commands/playlists.py]:
```suggestion
                            log.debug("Query is not allowed in %r (%d)", ctx.guild.name, ctx.guild.id)
```

File: redbot/cogs/audio/core/events/lavalink.py [hunk 19]
```diff
@@ -95,7 +92,7 @@ async def lavalink_event_handler(
             current_track, self.local_folder_current_path
         )
         status = await self.config.status()
-        log.debug(f"Received a new lavalink event for {guild_id}: {event_type}: {extra}")
+        log.debug("Received a new lavalink event for %d: %s: %s", guild_id, event_type, extra)
```
Comment (Jackenmen) [inline redbot/cogs/audio/core/events/lavalink.py]:
```suggestion
        log.debug("Received a new lavalink event for %d: %s: %r", guild_id, event_type, extra)
```

File: redbot/cogs/audio/core/events/lavalink.py [hunk 20]
```diff
@@ -335,8 +335,7 @@ async def _websocket_closed_handler(
                 player.store("resumes", player.fetch("resumes", 0) + 1)
                 await player.resume(player.current, start=player.position, replace=True)
                 ws_audio_log.info(
-                    f"Player resumed in channel {channel_id} in guild: {guild_id} | "
-                    f"Reason: Error code {code} & {reason}."
+                    "Player resumed Reason: Error code %d & %s, %r", code, reason, player
```
Comment (Jackenmen) [inline redbot/cogs/audio/core/events/lavalink.py]:
```suggestion
                    "Player resumed | Reason: Error code %d & %s, %r", code, reason, player
```

File: redbot/cogs/audio/core/events/lavalink.py [hunk 21]
```diff
@@ -349,12 +348,21 @@ async def _websocket_closed_handler(
...
+                    guild_id,
+                    voice_ws.socket._closing or voice_ws.socket.closed,
+                    code,
+                    by_remote,
+                    reason,
+                    player,
                 )
                 ws_audio_log.debug(
-                    f"Reconnecting to channel {channel_id} in guild: {guild_id} | {delay:.2f}s"
+                    "Reconnecting to channel %d in guild: %d | %d:.2fs",
```
Comment (Jackenmen) [inline redbot/cogs/audio/core/events/lavalink.py]:
```suggestion
                    "Reconnecting to channel %d in guild: %d | %.2fs",
```

File: redbot/cogs/audio/core/events/lavalink.py [hunk 22]
```diff
@@ -433,16 +444,17 @@ async def _websocket_closed_handler(
                     player._con_delay = ExponentialBackoff(base=1)
                     delay = player._con_delay.delay()
                 ws_audio_log.debug(
-                    f"Reconnecting to channel {channel_id} in guild: {guild_id} | {delay:.2f}s"
+                    "Reconnecting to channel %d in guild: %d | %d.2fs", channel_id, guild_id, delay
```
Comment (Jackenmen) [inline redbot/cogs/audio/core/events/lavalink.py]:
```suggestion
                    "Reconnecting to channel %d in guild: %d | %.2fs", channel_id, guild_id, delay
```

File: redbot/cogs/audio/core/events/lavalink.py [hunk 23]
```diff
@@ -452,24 +464,29 @@ async def _websocket_closed_handler(
...
                     ws_audio_log.info(
                         "Voice websocket reconnected "
-                        f"to channel {channel_id} in guild: {guild_id} | "
-                        f"Reason: Error code {code} & Not playing."
+                        "to channel %d in guild: %d | "
+                        "Reason: Error code %d & Not playing.",
+                        channel_id,
+                        guild_id,
+                        code,
                     )
```
Comment (Jackenmen) [inline redbot/cogs/audio/core/events/lavalink.py]: Should the player be logged here too?

File: redbot/cogs/audio/core/tasks/startup.py [hunk 24]
```diff
@@ -208,7 +212,9 @@ async def restore_players(self):
                         tries += 1
                     except Exception as exc:
                         tries += 1
-                        debug_exc_log(log, exc, "Failed to restore music voice channel")
+                        debug_exc_log(
+                            log, exc, "Failed to restore music voice channel (%s)", vc_id
```
Comment (Jackenmen) [inline redbot/cogs/audio/core/tasks/startup.py]: Consistency with the same log above
```suggestion
                            log, exc, "Failed to restore music voice channel %s", vc_id
```

File: redbot/cogs/audio/core/utilities/formatting.py [hunk 25]
```diff
@@ -167,7 +167,7 @@ async def _search_button_action(
             query_obj=query,
         ):
             if IS_DEBUG:
-                log.debug(f"Query is not allowed in {ctx.guild} ({ctx.guild.id})")
+                log.debug("Query is not allowed in %s (%d)", ctx.guild, ctx.guild.id)
```
Comment (Jackenmen) [inline redbot/cogs/audio/core/utilities/formatting.py]:
```suggestion
                log.debug("Query is not allowed in %r (%d)", ctx.guild.name, ctx.guild.id)
```

File: redbot/cogs/audio/core/utilities/player.py [hunk 26]
```diff
@@ -452,7 +452,7 @@ async def _enqueue_tracks(
                     query_obj=query,
                 ):
                     if IS_DEBUG:
-                        log.debug(f"Query is not allowed in {ctx.guild} ({ctx.guild.id})")
+                        log.debug("Query is not allowed in %s (%d)", ctx.guild, ctx.guild.id)
```
Comment (Jackenmen) [inline redbot/cogs/audio/core/utilities/player.py]:
```suggestion
                        log.debug("Query is not allowed in %r (%d)", ctx.guild.name, ctx.guild.id)
```

File: redbot/cogs/audio/core/utilities/player.py [hunk 27]
```diff
@@ -544,7 +542,7 @@ async def _enqueue_tracks(
...
```
Comment (Jackenmen) [inline redbot/cogs/audio/core/utilities/player.py]:
```suggestion
                        log.debug("Query is not allowed in %r (%d)", ctx.guild.name, ctx.guild.id)
```

File: redbot/cogs/audio/core/utilities/playlists.py [hunk 28]
```diff
@@ -428,14 +428,14 @@ async def _load_v2_playlist(
 
                 track = result.tracks[0]
             except Exception as err:
-                debug_exc_log(log, err, f"Failed to get track for {song_url}")
+                debug_exc_log(log, err, "Failed to get track for %s", song_url)
```
Comment (Jackenmen) [inline redbot/cogs/audio/core/utilities/playlists.py]:
```suggestion
                debug_exc_log(log, err, "Failed to get track for %r", song_url)
```

File: redbot/cogs/audio/core/utilities/playlists.py [hunk 29]
```diff
@@ -428,14 +428,14 @@ async def _load_v2_playlist(
...
                 continue
             try:
                 track_obj = self.get_track_json(player, other_track=track)
                 track_list.append(track_obj)
                 successful_count += 1
             except Exception as err:
-                debug_exc_log(log, err, f"Failed to create track for {track}")
+                debug_exc_log(log, err, "Failed to create track for %s", track)
```
Comment (Jackenmen) [inline redbot/cogs/audio/core/utilities/playlists.py]:
```suggestion
                debug_exc_log(log, err, "Failed to create track for %r", track)
```

**Review by Jackenmen — CHANGES_REQUESTED** [review]

File: redbot/cogs/audio/core/events/lavalink.py (hunk 23 above)
Comment (Drapersniper) [reply redbot/cogs/audio/core/events/lavalink.py]: Yes, Added

**Review by Jackenmen — DISMISSED** [review]
Looks good to me, waiting for RLL 0.8.1.

**Review by Jackenmen — APPROVED** [review]
Looks good to me

**Pull request merged**